From 164334361b94aff2db79918125b5c9b53d7f4b1a Mon Sep 17 00:00:00 2001
From: Corentin LABBE <clabbe@baylibre.com>
Date: Tue, 6 Aug 2019 13:11:30 +0200
Subject: [PATCH] devce-type: Fix meson-g12a-sei510

This patch fix lots of problem found on meson-g12a-sei510
- Fix corruption of uboot commands
- Fix boot action, since issuing just "boot" led to ignore what LAVA
setupped via TFTP
---
 .../device-types/meson-g12a-sei510.jinja2     | 23 ++++++++++---------
 1 file changed, 12 insertions(+), 11 deletions(-)

diff --git a/lava_scheduler_app/tests/device-types/meson-g12a-sei510.jinja2 b/lava_scheduler_app/tests/device-types/meson-g12a-sei510.jinja2
index 34b30c75d..d3641a75a 100644
--- a/lava_scheduler_app/tests/device-types/meson-g12a-sei510.jinja2
+++ b/lava_scheduler_app/tests/device-types/meson-g12a-sei510.jinja2
@@ -1,16 +1,12 @@
 {# device_type = "meson-g12a-sei510" #}
 {% extends 'meson-g12-common.jinja2' %}
-{% set base_kernel_args = base_kernel_args|default('') %}
-{% set uboot_bootx_cmd = 'boot' %}
-{% set base_uboot_bootcmd = uboot_bootcmd|default(
- "          - '" + uboot_bootx_cmd|default('{BOOTX}')) + "'" -%}
-{% set baud_rate = baud_rate|default('115200') %}
-{% set boot_character_delay = boot_character_delay|default(5) %}
+
+{% set uboot_mkimage_arch = 'arm64' %}
+
 {% set usb_uuid = usb_uuid|default('') %}
 {% set usb_device_id = usb_device_id|default('') %}
-{% set bootloader_prompt = bootloader_prompt|default('=>') %}
-{% set interrupt_prompt = interrupt_prompt|default('Hit any key to stop autoboot') %}
 {% set interrupt_char = interrupt_char|default(' ') %}
+
 {% set adb_serial_number = adb_serial_number|default("'0000000000'") %}
 {% set fastboot_serial_number = fastboot_serial_number|default("0000000000") %}
 {% set device_info = device_info|default([{'board_id': fastboot_serial_number}]) %}
@@ -21,16 +17,21 @@
         mmc:
           commands:
           - setenv console ttyUSB0,115200n8
-          - " + base_uboot_bootcmd + "
 " %}
-{% set root_device = root_device | default('/dev/ttyAML0') %}
+
 {% set kernel_args = kernel_args | default("consoleblank=0 earlycon") %}
+
 {% set booti_kernel_addr = booti_kernel_addr | default('0x01080000') %}
-{% set booti_ramdisk_addr = booti_ramdisk_addr | default('0x6000000') %}
+{% set booti_ramdisk_addr = booti_ramdisk_addr | default('0x8000000') %}
 {% set booti_dtb_addr = booti_dtb_addr | default('0x1000000') %}
+
 {% set bootz_kernel_addr = booti_kernel_addr %}
 {% set bootz_ramdisk_addr = booti_ramdisk_addr %}
 {% set bootz_dtb_addr = booti_dtb_addr %}
+
 {% set bootm_kernel_addr = bootz_kernel_addr %}
 {% set bootm_ramdisk_addr = bootz_ramdisk_addr %}
 {% set bootm_dtb_addr = bootz_dtb_addr %}
+
+{% set interrupt_ctrl_list = ['c'] %}
+
-- 
2.21.0

